javascript/builtins/WeakMap.json

```diff
@@ -51,7 +51,7 @@
               "version_added": "8"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": false
```

Comment:
WeakMap is probably supported 
Amazed by our enhanced linter catching this! :100: @vinyldarkscratch

Reply:
Nice catch!!